classes/CurseTwitter.php

```diff
@@ -164,16 +136,11 @@ public function setTweetCount($maximum) {
 
 	/**
 	 * Check for type of list.
-	 *
-	 * @access	public
-	 * @param	array	args
-	 * @param	string	$input
-	 * @return	string	$type
 	 */
-	public function setType(&$args, $input) {
-		$type     = isset($args['type']) ? $args['type'] : null;
-		$slug     = isset($args['list']) ? $args['list'] : null;
-		$widgetid = isset($args['widgetid']) ? $args['widgetid'] : null;
+	public function setType(array &$args, string $input) {
```

Comment:
```suggestion
	public function setType(array $args, string $input) {
```
seems like we don't need it here either